src/WakuRlnV2.sol

```diff
@@ -133,34 +130,66 @@ contract WakuRlnV2 is Initializable, OwnableUpgradeable, UUPSUpgradeable {
         return rateCommitment != 0;
```

Comment:
It looks like this is the only usage of the function `idCommitmentToMetadata`, and we ignore two out of three values it returns. Would it be more efficient to implement something like `getRateCommitment` instead?

Reply:
Hmm indeed. We could probably just create a modifier that does `if (members[idCommitment].userMessageLimit != 0) revert DuplicateIdCommitment()` and use that in both `register` functions.

Reply:
Fixed in https://github.com/waku-org/waku-rlnv2-contract/pull/13/commits/522599c704702856e41a7abc5e2dba8ef2c0fb3d